Lib/glyphsLib/builder/smart_components.py

```diff
@@ -0,0 +1,87 @@
+import pickle
```

Comment:
is there any documentation about "smart components" that we can link to, perhaps in a module docstring here?
All this looks like dark magic to me, let alone to a casual passer-by or our future selves